lib/ansible/executor/module_common.py

```diff
@@ -372,7 +393,10 @@ def atexit_coverage():
 
 ANSIBALLZ_COVERAGE_CHECK_TEMPLATE = '''
         try:
-            imp.find_module('coverage')
+            if PY3:
+                importlib.util.find_spec('coverage')
```

Comment:
This will return `None` if it is not found, so we'll need to test that and raise an exception.

See https://docs.python.org/3/library/importlib.html#importlib.util.find_spec

Reply:
Addressed